Bizyhood/Public/js/bizyhood-custom.js

```diff
@@ -1,4 +1,59 @@
 jQuery(function($){
+  
+  if (jQuery('#question_cta_form').length > 0) {
+    
+    console.log('form');
+    
+    var bizyhood_id = jQuery('#question_cta_form').attr('data-business');
+    
+    jQuery("#question_cta_form").submit(function(e){
+      e.preventDefault();
+      var fname   = jQuery('#question_cta_modal #question_cta_fname').val();
+      var lname   = jQuery('#question_cta_modal #question_cta_lname').val();
+      var email   = jQuery('#question_cta_modal #question_cta_email').val();
+      var message = jQuery('#question_cta_modal #question_cta_message').val();
+      
+      var data = {
+        'action': 'question_cta',
+        'bizyhood_id': bizyhood_id,
+        'text': message,
+        'email': email,
+        'first_name': fname,
+        'last_name': lname,
+      };
+      // We can also pass the url value separately from ajaxurl for front end AJAX implementations
+      jQuery.post(ajax_object.ajax_url, data, function(response) {
+        console.log('Got this from the server: ');
+        console.log(response);
+        if(response.error == true) {
+          jQuery('.response_area').addClass('bh_error').html(response.message);
+          
+          setTimeout(function () {
+            jQuery('.response_area').removeClass('bh_error').html('');
+          }, 10000);
+          
+        } else {
+          if (response.code == 200 || response.code == 201) {
+            jQuery('.response_area').text('Thank you for posting your inquiry! We will get back to you asap.');
```

Comment:
Let's make this text:

Thank you for your question. We will get back to you as soon as possible.

Reply:
Changing and submitting.